samples/cellular/nrf_cloud_multi_service/sysbuild.conf

```diff
@@ -1,8 +1,8 @@
 #
-# Copyright (c) 2023 Nordic Semiconductor
+# Copyright (c) 2024 Nordic Semiconductor
```

Comment:
I think we talked about changing the year, it doesn't matter much to change from 23 to 24 since the copyright is good for ~50 years. not a blocker though.